src/style.css

```
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
```

```suggestion
  font-weight: 800;
```
it can also help to fix tests
also set font-family for html selector

---

I tried to apply all advices but have the same problem with tests. Also, when set font-family for html I got this error message from Stylelint extension: `Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)`